docs_sources/erp-eml-epa-notes.adoc

```diff
@@ -0,0 +1,130 @@
+* link:https://simplifier.net/erezept-workflow/gem_erp_pr_medicationdispense[GEM_ERP_PR_MedicationDispense]
+* link:https://simplifier.net/erezept-workflow/gem_erp_pr_medication[GEM_ERP_PR_Medication]
+
+Instanzen, die zu diesen Profilen konform sind, sind damit auch für die Kommunikation mit der ePA geeignet. Auf dieser Seite sollen Besonderheiten und Hilfestellungen gegeben werden.
+
+== Erzeugen des Abgabedatensatzes
+
+Die API für die $close-Operation am E-Rezept-Fachdienst wird verändert. Bisher wurde das MedicationDispense Objekt direkt übergeben. Ab dem 15.01.2025 wird bei Verwendung der neuen Profile gefordert, dass die Informationen in einem Parameters Objekt übergeben werden:
+
+image:parameters-schematics.png[width=50%]
+
+Die Medication ist damit nicht mehr in der MedicationDispense eingebettet, sondern wird als eigenes Objekt übergeben und von der MedicationDispense referenziert.
+
+Im folgenden ein Beispiel für die Übermittlung eines Abgabedatensatzes mit den neuen Profilen:
+
+//TODO: Change to real example from Examples Repository
+.Beispiel für eine Abgabe mit Parameters (Klicken zum Ausklappen)
+[%collapsible]
+====
+[source,xml]
+----
+include::../resources/temp/Parameters-ExampleCloseInputParameters.xml[]
+----
+====
+
+Bei einer Mehrfachabgabe werden die Medication und MedicationDispense Objekte alle im Parameters Objekt übergeben.
+.Beispiel für eine Mehrfachabgabe mit Parameters (Klicken zum Ausklappen)
+[%collapsible]
+====
+
+Schematische Darstellung der Struktur:
+image:parameters-schematics-multiple.png[width=50%]
+//TODO: Change to real example from Examples Repository
+[source,xml]
+----
+include::../resources/temp/Parameters-ExampleDispenseInputParametersMultipleMedicationDispenses.xml[]
+----
+====
+
+Diese Änderung geschieht in Vorausschau auf die Abgabe von OTCs gegenüber dem ePA-Aktensystem, was die gleiche Struktur erfordern wird.
+
+=== Erzeugen von Freitext-, PZN- und Wirkstoff-Medications
+
+Neu ab dem 15.01.2025 ist, dass abgebende Systeme statt einer Medication die der KBV-Profile entspricht, eine Medication verwenden, die dem GEM_ERP_PR_Medication Profil entspricht. Dieses Profil wurde von link:https://simplifier.net/epa-medication/epamedication[EPAMedication] abgeleitet und ist damit auch für die Kommunikation mit der ePA geeignet.
+
+Für das Erzeugen von Freitext-, PZN- und Wirkstoff-Medications können die KBV-Profile auf das neue Profil gemappt werden. Hierzu gibt es eine Mappingtabelle, die aufzeigt welche Werte aus den KBV-Profilen in das neue Profil übernommen werden können und an welche Stelle sie zu setzen sind: link:https://gematik.github.io/api-erp/erp_epa_mapping_details/KBV_PR_ERP_Medication_Compounding%7C1.1.0_KBV_PR_ERP_Medication_FreeText%7C1.1.0_KBV_PR_ERP_Medication_Ingredient%7C1.1.0_KBV_PR_ERP_Medication_PZN%7C1.1.0_to_EPAMedication%7C1.0.2-rc1.html[Mappingtabelle für Medications].
```

Comment:
Hier müssen wir vorsichtig sein, da sich der Link noch mindestens zweimal ändern wird, da die URL die Version des Zielprofils enthält:
EPAMedication%7C1.0.2-rc1.html

Wir haben drei Optionen:

1. Immer daran denken, diesen Link zu aktualisieren, wenn die Mapping-Tabelle aktualisiert wird.
2. Die Links und damit die Dateinamen unter https://github.com/gematik/api-erp/tree/feature/erp_epa_mapping_details/docs/erp_epa_mapping_details und in https://github.com/gematik/api-erp/blob/feature/erp_epa_mapping_details/docs/erp_epa_mapping_details/README.md generischer gestalten. (Bricht die Namingconverntion des Verwendeten Toolings- keine Copy and Paste mehr möglich)
3. Auf die Übersichtsseite verlinken und beschreiben, welchen Link man wählen muss: https://gematik.github.io/api-erp/erp_epa_mapping_details/


Ich denke, Option 2 ist die beste, da man, falls man den Link speichert, immer auf der aktuellen Version landet.

Reply:
Ich habe die Links der Mapping Tabellen so angepasst, das sie  zwar die Version des Packages aber nicht den ReleaseCandidate im Namen tragen:
https://gematik.github.io/api-erp/erp_epa_mapping_details/